src/ui/ui.css

```diff
@@ -56,7 +58,6 @@ swg-loading {
   position: absolute !important;
   top: 40% !important;
```

Comment:
Why then keep `top`/`left`? I don't think the content will ever be in the middle this way...